src/components/auth/utils/currencies/CurrenciesItems.js

```diff
@@ -35,7 +35,7 @@ const CurrencyField = styled(TextField)((props) => ({
 }));
 
 function CurrenciesItems({ names, currency, setCurrency }) {
-  let results = [];
+  const results = [];
```

Comment:
Changing `let` to `const` is a good thing here, but you should really be using `names.map`, like:

```
const results = Object.values(names).map((currencyName) => {
  return (
    <MenuItem key={currencyName} value={currencyName}>
      {currencyName}
    </MenuItem>
  )
})
```